src/networkmanager.cpp

```diff
@@ -67,10 +67,7 @@ QByteArray NetworkManager::userAgent() {
   {
     QStringList flags;
     flags.append(QString("sys:") + NetworkManager::osVersion());
-
-    if (Feature::get(Feature::Feature_inAppPurchase)->isSupported()) {
-      flags.append("iap:true");
-    }
+    flags.append("iap:true");
```

Comment:
It would be nice if we can get rid of this flag and use just the client version in Guardian

Reply:
This is a good idea, but not small, i've opened https://mozilla-hub.atlassian.net/browse/VPN-3248.